### DISCUSSION TITLE
Will there any way to make the Full CRUD operations using wp-data and redux way available here ?

### DISCUSSION THREAD

**Discussion by maniruzzamanak** [ManiruzzamanAkash/wp-react-kit#14] [Q&A] [answered]
Hi, Akash,
My one question - Currently you're using Wp-Data and redux.

But is there any way to make the Full CRUD operations using wp-data and redux way available here ?

**Comment by ManiruzzamanAkash** [comment, marked as answer]
Hi, @maniruzzamancse 

Yes, there is WP-data support fully. Currently we've added List, Search, Delete stuffs. Create, Edit is pending.
You can make the Pull-request to do that.

Also that's in our next release.
